packages/nouns-contracts/contracts/governance/NounsDAOInterfaces.sol

```diff
@@ -488,10 +488,11 @@ contract NounsDAOStorageV3 {
         // ================ V3 ================ //
         uint256 lastMinuteWindowInBlocks;
         uint256 objectionPeriodDurationInBlocks;
-        mapping(uint256 => bool) proposeBySigNonces;
         /// @notice The proposal at which to start using `startBlock` instead of `creationBlock` for vote snapshots
         /// @dev To be zeroed-out and removed in a V3.1 fix version once the switch takes place
         uint256 voteSnapshotBlockSwitchProposalId;
+        /// @notice signatures already used that shouldn't be used again
+        mapping(bytes32 => bool) usedSigs;
```

Comment:
@davidbrai I think we should move this variable up, so `voteSnapshotBlockSwitchProposalId` is the last variable in storage layout, so we can easily zero it and delete it once we're done with the snapshot block migration.

wdyt?

Reply:
done